public/javascript/index.js

```diff
@@ -1,23 +1,132 @@
-const charactersAPI = new APIHandler('http://localhost:8000');
+const charactersAPI = new APIHandler('https://minions-api.herokuapp.com/');
+
+updateList = () => {
+  charactersAPI
+    .getFullList()
+    .then(response => {
+      const { data } = response
+
+      let currentDiv = document.querySelector(".characters-container");
+
+      while (currentDiv.firstChild) {
+        currentDiv.removeChild(currentDiv.firstChild);
+      }
+
+      data.forEach(elm => {
+        charactersAPI.writtenBlock(elm, currentDiv)
+
+      }
+      )
+
+    })
+    .catch(err => console.log('error', err))
+
+}
 
 window.addEventListener('load', () => {
   document.getElementById('fetch-all').addEventListener('click', function (event) {
+    updateList()
 
   });
 
   document.getElementById('fetch-one').addEventListener('click', function (event) {
 
+    event.preventDefault()
+    const inputs = document.querySelector('.operation input')
+
+    charactersAPI
+      .getOneRegister(inputs.value)
+      .then(selection => {
+        let div = document.querySelectorAll(".operation")[1];
+        charactersAPI.writtenBlock(selection, div)
+      })
```

Comment:
Esto genera una caja de undefineds.
```suggestion
      .then(response => {
        let div = document.querySelectorAll(".operation")[1];
        charactersAPI.writtenBlock(response.data, div)
      })
```